src/main/java/by/onliner/core/listener/TestListener.java

```diff
@@ -159,4 +168,30 @@ public void onTestFailedWithTimeout(ITestResult result) {
             return new byte[0];
         }
     }
+
+    private String getBrowser() {
```

Comment:
1. Are you sure it's needed? what does BrowserConfig.getType() returns?
2. Are you sure this is the functionality of TestListener